<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>471</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo general del fondo */
QWidget#Login {
    background-color: #F5F5F5;
}

/* Estilo del título &quot;Iniciar Sesión&quot; */
QLabel#Loginlabel {
    background-color: #85BB65;
    color: white;
    font-size: 20px;
    font-weight: bold;
    border-radius: 8px;
    padding: 8px;
}

/* Estilo de los labels */
QLabel {
    font-size: 14px;
    color: #333333;
}

/* Estilo de los campos de entrada */
QLineEdit {
    border: 2px solid #85BB65;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
}
QLineEdit:focus {
    border: 2px solid #6FA24C;
}

/* Estilo de los botones */
QPushButton {
    background-color: #85BB65;
    color: white;
    font-size: 16px;
    font-weight: bold;
    border-radius: 5px;
    padding: 8px;
}
QPushButton:hover {
    background-color: #6FA24C;
}
QPushButton:pressed {
    background-color: #5C8E3E;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>502</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ColumnaPrincipal">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <widget class="QLabel" name="Loginlabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Registrarse</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Campos">
      <item>
       <widget class="QLabel" name="UsuarioLabell">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UsuarioEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="EmailLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EmailEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="ContrasnaLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Conraseña</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="RepeatPassword">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Repetir Conraseña</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RepeatPasswordEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Botones">
      <item>
       <widget class="QPushButton" name="BackButon">
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RegisterButtom">
        <property name="text">
         <string>Registrarse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
